digdag-server/src/main/java/io/digdag/server/rs/AttemptResource.java

```diff
@@ -53,6 +53,8 @@
     private final AttemptBuilder attemptBuilder;
     private final WorkflowExecutor executor;
     private final ConfigFactory cf;
+    private static final int MAX_ATTEMPTS_PAGE_SIZE = 1000;
```

Comment:
What do you think that we'd better to extract the max page size as system config? The constant value may fit with someone. may not fit with others. For example, in my application, I want to keep using `100` as page size and max page size. But I believe that `1,000` looks reasonable to other users'.